projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/shipping-address/shipping-address.component.html

```diff
@@ -59,11 +59,21 @@ <h3 class="cx-checkout-title d-none d-lg-block d-xl-block">
     </ng-container>
 
     <ng-template #newAddressForm>
-      <cx-address-form
-        showTitleCode="true"
-        (submitAddress)="addNewAddress($event)"
-        (backToAddress)="hideNewAddressForm()"
-      ></cx-address-form>
+      <ng-container *ngIf="existingAddresses.length; else initialAddressForm">
+        <cx-address-form
+          showTitleCode="true"
+          (backToAddress)="hideNewAddressForm()"
```

Comment:
even though it's a default param we should pass false here for the sake of readability